wordpress/resources/app.yaml

```diff
@@ -0,0 +1,152 @@
+      - profile: front
+        count: 1
+      - profile: db
+        count: 1
+#HA Master nodes (3) configuration with a 2 fronts and a db
+    - name: "large"
+      description: "High Availability Master Node configuration with 2 Wordpress PHP nodes and 1 My"
+      nodes:
+      - profile: front
+        count: 2
+      - profile: db
+        count: 1
+      - profile: master
+        count: 3
+
+# This section allows to define what flavors of servers are required for
+# this cluster. This example uses a single flavor called "node" but you
+# can have flavors like "database" or "gpu", etc.
+#
+#
+nodeProfiles:
+  - name: general
+    description: "full node"
+    labels:
+      front: "true"
+      db: "true"
+    # gravity will validate that the nodes have the requested amounts
+    # of RAM/CPU
+    requirements:
+      cpu:
+        min: 2
+      ram:
+        min: "2GB"
+  - name: master
+    description: "master node"
+    labels:
+      node-role.kubernetes.io/master: "true"
+    # gravity will validate that the nodes have the requested amounts
+    # of RAM/CPU
+    requirements:
+      cpu:
+        min: 2
+      ram:
+        min: "2GB"
+  - name: front
+    description: "Web front end node"
+    labels:
+      front: "true"
+      node-role.kubernetes.io/node: "true"
+    # gravity will validate that the nodes have the requested amounts
+    # of RAM/CPU
+    requirements:
+      cpu:
+        min: 2
+      ram:
+        min: "2GB"
+  - name: db
+    description: "Database node"
+    labels:
+      db: "true"
+      node-role.kubernetes.io/node: "true"
+    # gravity will validate that the nodes have the requested amounts
+    # of RAM/CPU
+    requirements:
+      cpu:
+        min: 2
+      ram:
+        min: "2GB"
+
+
+
+
+# This section allows you to specify Kubernetes jobs that will be executed
+# inside the cluster when certain cluster lifecycle events happen
+hooks:
+  install:
+    job: file://install.yaml
+#As new versions of wordpress are realized this upgrade hook will make s
+  update:
+    job: file://upgrade.yaml
+
+  # in this example the hooks below do not do anything. we're listing them here
+  # to demostrate the possibility of customizing cluster life cycle.
+
+  clusterProvision:
+    job: file://clusterProvision.yaml
+  nodesProvision:
+    job: file://nodesProvision.yaml
+  nodesDeprovision:
+    job: file://nodesDeprovision.yaml
+  clusterDeprovision:
+    job: file://clusterDeprovision.yaml
+
+systemOptions:
+  # The "runtime" is a very important field. It defines which flavor of
+  # Kubernetes/Docker/etc (and other components) will be packaged into
+  # the application bundle. Gravitational Inc is publishing and maintaining
+  # the list of battle-tested combinations (called "runtimes"). You can see
+  # the full list here:
+  #      https://gravitational.com/gravity/docs/changelog/
+  # .. or by executing 'tele ls' command
+  # If the "runtime" isn't set, it'll default to the systems tele binary.
```

Comment:
`tele` will choose the version of runtime automatically.

Reply:
Removed this comment.  It was copied over.  Added in comment on storage backend